### ISSUE TITLE
[BUG v2]: WAI-ARIA compliance issue WCAG 4.1.2

### ISSUE THREAD

**Issue by orestbida** [orestbida/cookieconsent#544]
### Discussed in https://github.com/orestbida/cookieconsent/discussions/542

<div type='discussions-op-text'>

<sup>Originally posted by **esginmurat** June 27, 2023</sup>
Hello thanks for cookieconsent. (@orestbida) WCAG 4.1.2 

I see that the aria-level tag specified in WCAG 4.1.2 is missing. (attached photo)

![aria roles problem](https://github.com/orestbida/cookieconsent/assets/17769013/8f1439a2-f084-4710-a1c2-912c3c2573ea)

Regards...</div>
